src/components/TrashData/TrashData.js

```diff
@@ -1,38 +1,22 @@
-import axios from "axios";
-import React, { useEffect, useReducer } from "react";
-import { trashReducer } from "../../utils/trashReducer";
+import React from "react";
+import { useNote } from "../../context/note-context";
 
 import { Notepad } from "../Notepad/Notepad";
 import { Searchbar } from "../SearchBar/Searchbar";
 
 export const TrashData = () => {
-  const api =
-    "https://my-json-server.typicode.com/Raghav888/mynoteappAPI/trash";
-
-  const [trashstate, trashdispatch] = useReducer(trashReducer, {
-    trashData: [],
-  });
-
-  useEffect(() => {
-    (async () => {
-      try {
-        const response = await axios.get(api);
-        trashdispatch({
-          type: "ADD_TRASH_DATA",
-          payload: { value: response.data },
-        });
-      } catch (error) {
-        console.log(error);
-      }
-    })();
-  }, []);
+  const { noteListState } = useNote();
```

Comment:
This is good. Just a suggestion for future use :

you can replace ```{ noteListState } ``` ```{noteListState: { trashData } }``` so for mapping you have  to use only ```trashData```